testplan/common/utils/match.py

```diff
@@ -285,19 +298,58 @@ def match(
                     break
 
             self.position = self.log_stream.position
+            if self._debug_info_e is None:
+                self._debug_info_e = (
+                    str(self.position),
+                    time.time(),
+                    _truncate_line(line),
```

Comment:
if we are reaching eof, more useful info would be the last line seen, rather than <eof>